package.json

```diff
@@ -45,6 +45,6 @@
   ],
   "dependencies": {
     "axios": "1.7.4",
-    "verus-typescript-primitives": "https://github.com/VerusCoin/verus-typescript-primitives.git"
+    "verus-typescript-primitives": "https://github.com/mcstoer/verus-typescript-primitives.git#getidentitycontent"
```

Comment:
Changes have been merged into typescript primitives now so all that needs to be done is an update of the existing lib, should only affect the yarn.lock file.

Reply:
Updated. Thank you!